### DISCUSSION TITLE
Reddit

### DISCUSSION THREAD

**Discussion by rinzwind5** [odin-lang/Odin#3202] [Ideas/Requests]
Please add the reddit odinlang to the odin community webpage. I really (really) dislike discord for tis chaotic unstructured adhoc format and much prefer forum like ordered discussions.never understood why discord is even a thing. Discord is only for the original person asking a specific question, while reddit is reusable for anyone having similar questions and followup discussions. And of course, discord is not even searchable through search engines.

**Comment by programandala-net** [comment]
I totally agree. Contrary to e-mail groups, web forums, Reddit... Discord does not allow ordered, reusable, searchable discussions. It's not designed for that.

**Comment by rinzwind5** [comment]
Maybe for the inner dev(s) of Odin, Discord is like a instant/realtime ad-hoc group chat to discuss various random dev issues right here and now and that is why its there. But it is unsuited for general reusable and decently archived support for the end-users of Odin. It is not documenting anything and very hard to find answers after the fact by third parties and missing chronologic grouped topic discussions (aka forum). Thanks for reading ;)  Imagine Github Discussions being like Discord, the horror.

**Comment by laytan** [comment]
I am pretty sure a PR that adds it to the website would be accepted
